src/musesounds/internal/musesoundsrepository.cpp

```diff
@@ -131,11 +131,16 @@ void MuseSoundsRepository::th_requestSounds(const UriQuery& soundsUri, std::func
         return;
     }
 
-    JsonDocument soundsInfoDoc = JsonDocument::fromJson(ByteArray::fromQByteArray(receivedData.data()));
-
     RetVal<SoundCatalogueInfoList> result;
-    result.ret = make_ret(Ret::Code::Ok);
-    result.val = parseSounds(soundsInfoDoc);
+
+    std::string err;
+    JsonDocument soundsInfoDoc = JsonDocument::fromJson(ByteArray::fromQByteArray(receivedData.data()), &err);
```

Comment:
There is the same problem in MuseScoreComService::downloadAccountInfo(), AbstractCloudService::readTokens(), AudioComService::downloadAccountInfo(). Shouldn't we fix it there as well?

Reply:
done